sapp/sarif.py

```diff
@@ -64,11 +65,26 @@ def __init__(
             for rule in tool_warning_messages:
                 rules_json.append({"id": str(rule.code), "name": rule.message})
             driver_json["rules"] = rules_json
+        elif self.tool == "mariana-trench":
+            driver_json["name"] = "Mariana Trench"
+            driver_json[
+                "informationUri"
+            ] = "https://github.com/facebook/mariana-trench/"
+
+            tool_warning_messages = get_warning_message_range(
+                session,
+                self._tool_warning_code_ranges[self.tool][0],
+                self._tool_warning_code_ranges[self.tool][1],
+            )
+            rules_json = []
+            for rule in tool_warning_messages:
+                rules_json.append({"id": str(rule.code), "name": rule.message})
+            driver_json["rules"] = rules_json
```

Comment:
Instead of copy/pasting this part from lines 59-67, could we move it outside of the if/elif?